parser.y

```diff
@@ -1383,7 +1384,12 @@ PlacementRole:
 PlacementCount:
 	"REPLICAS" "=" LengthNum
 	{
-		$$ = $3
+		cnt := $3.(uint64)
+		if cnt == math.MaxUint64 {
+			yylex.AppendError(yylex.Errorf("Invalid placement option REPLICAS, it is not allowed to be MaxUint64"))
+			return 1
+		}
+		$$ = cnt
```

Comment:
How about 0 stands for no declaration, and forbid it to be set to 0?

Reply:
Done